packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+
+/**
+ * @title AttestationStation
+ * @dev   Contract for creating attestations.
+ * @notice The AttestationStation contract is a contract for creating on chain attestations
+ *         It has a very simple interface for creating attestations.
+ *         This contract is not yet audited
+ */
+contract AttestationStation is Semver {
+    /**
+     * @notice Struct representing data that is being attested
+     *
+     * @custom:field about Address being attested about (not creator/msg.sender)
+     * @custom:field key   A bytes32 key for the attestation.
+     * @custom:field val   The attestation as arbitrary bytes
+     */
+    struct AttestationData {
+        address about;
+        bytes32 key;
+        bytes val;
+    }
+
+    /**
+     * @notice Maps addresses to attestations
+     * @dev addresses map to attestations map of
+     *      `about` addresses to key/values
+     *      key/values are a map of bytes32 to bytes
+     */
+    mapping(address => mapping(address => mapping(bytes32 => bytes))) public attestations;
+
+    /**
+     * @notice Emitted when Attestation is created
+     *
+     * @param creator Address that attested.
+     * @param about  Address attestation is about.
+     * @param key Key of the attestation.
+     * @param val Value of the attestation.
+     */
+    event AttestationCreated(
+        address indexed creator,
+        address indexed about,
```

Comment:
slight nit: I feel like `topic` is a better term than `about` here

Reply:
nice I agree!

Reply:
actually nvm don't want to change that since it's a breaking change to @gateway/contracts @gateway/backend and @gateway/frontend